h2/src/main/org/h2/command/Parser.java

```diff
@@ -4558,6 +4558,13 @@ private CreateSchema parseCreateSchema() {
         } else {
             command.setAuthorization(session.getUser().getName());
         }
+        if (readIf("WITH")) {
+            ArrayList<String> tableEngineParams = New.arrayList();
```

Comment:
Looks like a code duplication from `parseCreateTable`. Lets move this common part to separate method.